bootstrap/service.go

```diff
@@ -146,6 +146,23 @@ func (bs bootstrapService) Add(ctx context.Context, token string, cfg Config) (C
 		return Config{}, err
 	}
 
+	// Check for existing connection between channel and thing
+	// If it exists set state to acive
+	state := Inactive
+	pm := mgsdk.PageMetadata{}
+	for _, channel := range cfg.Channels {
+		tp, err := bs.sdk.ThingsByChannel(channel.ID, pm, token)
+		if err != nil {
+			return Config{}, errors.Wrap(svcerr.ErrMalformedEntity, err)
+		}
+		for _, thing := range tp.Things {
+			if thing.ID == cfg.ThingID {
+				state = Active
+				break
+			}
+		}
+	}
```

Comment:
@nyagamunene this will be resolve through a seperate endpoint here: https://github.com/absmach/magistrala/issues/2301